commands/inventory/add_agent_external_exporter.go

```diff
@@ -0,0 +1,103 @@
+// pmm-admin
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inventory
+
+import (
+	"github.com/percona/pmm/api/inventorypb/json/client"
+	"github.com/percona/pmm/api/inventorypb/json/client/agents"
+
+	"github.com/percona/pmm-admin/commands"
+)
+
+var addAgentExternalExporterResultT = commands.ParseTemplate(`
+External Exporter added.
+Agent ID              : {{ .Agent.AgentID }}
+Runs on Node ID       : {{ .Agent.RunsOnNodeID }}
+Service ID            : {{ .Agent.ServiceID }}
+Username              : {{ .Agent.Username }}
+Scheme                : {{ .Agent.Scheme }}
+Metric Path           : {{ .Agent.MetricPath }}
+Listen port           : {{ .Agent.ListenPort }}
```

Comment:
Sorry for this picky comment but Path, Node, ID words are capitalized but port and lables don't